assets/dev/js/editor/regions/navigator/element.js

```diff
@@ -218,20 +227,11 @@ export default class extends Marionette.CompositeView {
 	exitTitleEditing() {
 		this.ui.title.attr( 'contenteditable', false );
 
-		const settingsModel = this.model.get( 'settings' ),
-			oldTitle = settingsModel.get( '_title' ),
-			newTitle = this.ui.title.text().trim();
-
-		// When there isn't an old title and a new title, allow backbone to recognize the `set` as a change
-		if ( ! oldTitle ) {
-			settingsModel.unset( '_title', { silent: true } );
-		}
-
-		settingsModel.set( '_title', newTitle );
-
-		// TODO: Remove - After merge pull request #13605.
-		$e.internal( 'document/save/set-is-modified', {
-			status: true,
+		$e.run( 'document/elements/settings', {
```

Comment:
Here you should validate that the title is not empty

Reply:
The hook created because it is doing abnormal behavior, and on empty title it will return to the original title which is not what was before but the model title, maybe the name oh the hook `navigator-prevent-empty-title` is not right meaning?